docs/getting_started_ee/run_execution_environment.md

```diff
@@ -82,6 +81,21 @@ Execute a playbook inside the `postgresql_ee` EE against a remote host machine a
     ansible-navigator run test_remote.yml -i inventory --execution-environment-image postgresql_ee:latest --mode stdout --pull-policy missing --enable-prompts -u student -k -K
     ```
 
+## SSH keys forwarding
+
+If you run Ansible with SSH-keys-based authentication to avoid authentication prompts, use `ssh-agent` to forward the SSH keys to your EE:
```

Comment:
```suggestion
If you run Ansible with SSH-key-based authentication to avoid authentication prompts, use `ssh-agent` to forward the SSH keys to your EE:
```